app/views/layouts/admin/_sidebar.html.erb

```diff
@@ -0,0 +1,66 @@
+<ul id="accordionSidebar"
```

Comment:
I think it could be better to just have bigger menu items with bigger icons, but a lower number. Let me know what you think.
![image](https://user-images.githubusercontent.com/8007722/72658298-f5c0c280-397c-11ea-9025-d0497695a2f6.png)

Maybe the following:

- Instances
- Billing
- Account
- Support

Reply:
Updated. Using the collapsed version and increased the icons size. Let me know what you think.

Reply:
could the menu item text be bigger ? It seems small